Pd 70 build cli reference guide structure (#1791)
truenas/documentation: content/SCALE/SCALETutorials/TrueNASCLIShell/System/_index.md

---
title: "System"
geekdocCollapseSection: true
description: "This article describes how to configure the system in the TrueNAS CLI Shell." 
weight: 10
---

{{< toc >}}

## Performing Manual Updates

> To perform a manual update via the TrueNAS CLI Shell, you will first have to upload a manual update file onto the system.

Reviewer: past tense, make it present tense, active voice
```suggestion
To perform a manual update via the TrueNAS CLI Shell, you must first upload a manual update file onto the system.
```
